docs/config.md

```diff
@@ -0,0 +1,163 @@
+# Configuration instruction
+
+You can see the default configuration [here](../devtools/config/docker_compose_config.toml).
+
+## Basic configuration
+
+### `center_id`
+
+The center id of the server. This is used to generate the distributed unique ID through snowflake algorithm.
+
+type: `u16`
+
+### `machine_id`
+
+The machine id of the server. This is used to generate the distributed unique ID through snowflake algorithm.
+
+type: `u16`
+
+### `indexer_mode`
+
+If mercury is in the indexer mode, mercury will pull blocks from the CKB node and append the block data to the database. Otherwise, mercury will only handle RPC requests and do not append the block data.
```

Comment:
I suggest using `sync_mode` instead of `indexer_mode`, because the core function of this set is to sync or not to sync, which is much more understandable.

Reply:
If mercury is in the indexer mode, it will pull blocks from the CKB nodes and append the block data to the database.